pkg/scaffold/v1/webhook/admissionwebhooks.go

```diff
@@ -25,7 +25,7 @@ import (
 	"sigs.k8s.io/kubebuilder/pkg/scaffold/resource"
 )
 
-var _ input.File = &Server{}
+var _ input.File = &AdmissionWebhooks{}
```

Comment:
Following my checks and observations
- The change here in the interface shows correctly. 
- We can confirm that this file is scaffold by default for v1 projects. See [here](https://github.com/kubernetes-sigs/kubebuilder/search?q=%2F%2F+Builders+contain+admission+webhook+builders&unscoped_q=%2F%2F+Builders+contain+admission+webhook+builders). 
- So, it has been checked by the CI which is passing and the scaffold files were not impacted or changed. 

Then, it shows OK to be merged. However, since the v1 is deprecated I am not sure if we should or not move forward with changes there unless they are strictly required.  I will leave it with @mengqiy and/or @droot.